samples/android/build.gradle

```diff
@@ -0,0 +1,29 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
+buildscript {
+    ext.kotlin_version = '1.2.60'
```

Comment:
1.2.61

Reply:
I'd keep it for now, will update once I get `1.x` and `2.x` branches updated.